www/content/tutorial.md

```diff
@@ -820,18 +820,29 @@ Result.isOk (List.get ["a", "b", "c"] 1)
 ```
 
 ```roc
-Result.try (Str.toU64 "2") \idx -> List.get ["a", "b", "c", "d"] idx
```

Comment:
I expect this will be hard to parse for beginners, I tried a formatting change, but I think the separate `listGet` function is best in this case.